pkg/cliplugins/claims/plugin/edit.go

```diff
@@ -0,0 +1,193 @@
+	// The command will list those open claims and update `spec.AcceptableClaims` with the status based on the input of the user.
+
+	apibindings := []apiv1alpha1.APIBinding{}
+	// List permission claims for all bindings in current workspace.
+	if e.allBindings {
+		bindings, err := kcpClusterClient.Cluster(currentClusterName).ApisV1alpha1().APIBindings().List(ctx, metav1.ListOptions{})
+		if err != nil {
+			return fmt.Errorf("error listing apibindings in %q workspace: %w", currentClusterName, err)
+		}
+		apibindings = append(apibindings, bindings.Items...)
+	} else {
+		binding, err := kcpClusterClient.Cluster(currentClusterName).ApisV1alpha1().APIBindings().Get(ctx, e.APIBindingName, metav1.GetOptions{})
+		if err != nil {
+			return fmt.Errorf("error finding apibinding: %w", err)
+		}
+		apibindings = append(apibindings, *binding)
+	}
+
+	if len(apibindings) == 0 {
+		_, err = fmt.Fprintf(e.Out, "No apibindings available in %s.", currentClusterName)
+		if err != nil {
+			return err
+		}
+		return nil
+	}
+
+	_, err = fmt.Fprintf(e.Out, "Running interactive prompt. Enter (%s/%s/%s)\n", "Yes", "No", "Skip")
+	if err != nil {
+		return err
+	}
+
+	allErrors := []error{}
+	for _, apibinding := range apibindings {
+		openClaims := getOpenClaims(apibinding.Status.ExportPermissionClaims, apibinding.Spec.PermissionClaims)
+		if len(openClaims) == 0 {
+			_, err = fmt.Fprintf(e.Out, "No open claims found for apibinding: %q\n", apibinding.Name)
+			if err != nil {
+				allErrors = append(allErrors, err)
+			}
+		}
+
+		// List open claims and update `spec.AcceptableClaims` with the status based on the input of the user.
+		for _, openClaim := range openClaims {
+			// print the prompt and infer the user input.
+			action, err := getRequiredInput(e.In, e.Out, apibinding.Name, openClaim.Group, openClaim.Resource)
+			if err != nil {
+				allErrors = append(allErrors, err)
+			}
+
+			// Update apibinding based on the user input.
+			err = updateAPIBinding(ctx, action, openClaim, &apibinding, currentClusterName, kcpClusterClient)
+			if err != nil {
+				allErrors = append(allErrors, err)
+			}
+		}
+	}
+	return utilerrors.NewAggregate(allErrors)
+}
+
+// getOpenClaims gets the claims which are in a set of `exportClaims` but not in `spec.AcceptableClaims` of the apibinding.
+func getOpenClaims(exportClaims []apiv1alpha1.PermissionClaim, aceptableClaims []apiv1alpha1.AcceptablePermissionClaim) []apiv1alpha1.PermissionClaim {
+	openPermissionClaims := []apiv1alpha1.PermissionClaim{}
+
+	// Find the subset which is in exportClaims but not in AcceptableClaims.
+	exportClaimsMap := map[apiv1alpha1.GroupResource]apiv1alpha1.PermissionClaim{}
+	for _, e := range exportClaims {
+		exportClaimsMap[e.GroupResource] = e
+	}
+
+	acceptableClaimsMap := map[apiv1alpha1.GroupResource]apiv1alpha1.PermissionClaim{}
```

Comment:
These are not "acceptable" - they are already accepted, right?

Reply:
not necessarily accepted. They have a `AcceptableClaimState` (https://pkg.go.dev/github.com/kcp-dev/kcp/pkg/apis/apis/v1alpha1#AcceptablePermissionClaimState), where it can be in a rejected state too.